### ISSUE TITLE
Document path to settings files

### ISSUE THREAD

**Issue by ror3d** [statiqdev/statiqdev.github.io#45]
It might be helpful to add a note to input/framework/configuration/settings.md about how the settings files are searched for starting in the working directory of the application, *not* in your Engine's RootPath.

My setup right now has a bit of an upside-down layout, my engine.FileSystem.RootPath is something like "../", as I have the dotnet application in a child directory.

**Comment by daveaglick** [comment]
Good call! Right now I'm doing some major refactoring of the Statiq site and docs, so I'm just logging documentation improvements as issues. Once I've got that more stable, I'll start catching up on docs issues like this one. Thanks for the suggestion!

**Comment by ror3d** [comment]
Awesome. Would you prefer me to submit issues like this for this kind of thing, or to bring it up in the discussion? coming at the framework without much knowledge of C# or .NET or anything, I find myself with many things I would like more easily accessible

**Comment by daveaglick** [comment]
Either is fine! I'll move things around if needed. In general, I tend to think of issues as being actionable (a new feature request, a bug that needs fixing, documentation to write) and discussions as being more about examples, discussions, questions, etc. (which may result in an issue as well).

I'd say in your case (which is really valuable insight) - if it's something you've figured out and think it could use better documentation or examples for other folks, I'd say it's an issue. On the other hand, if it's something you're trying to figure out and could use some help with, open a discussion. Doesn't matter that much though, I'm around either way 😂
